pkg/template/api/v1/types.go

```diff
@@ -63,4 +63,8 @@ type Parameter struct {
 
 	// Optional: Indicates the parameter must have a value.  Defaults to false.
 	Required bool `json:"required,omitempty" description:"indicates the parameter must have a non-empty value or be generated"`
+
+	// Optional: Indicates clients should expect the user to potentially provide a multiline value for this parameter value.
+	// Defaults to false
+	Multiline bool `json:"multiline,omitempty"`
```

Comment:
How soon are we going to rev a v2 and how soon are we going to get more hints?  If we think we're going to have a multiple hints, I think I might want to have them segregated as a substruct.